### PR TITLE
Describe load and performance parameters in the README

### PR DESCRIPTION
**Pull request by hariso**
### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/streaming-benchmarks/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,21 @@ in Conduit is measured from when it's read from a source until it's ack-ed by al
...
+### Load parameters
+
+The following load parameters are used in tests:
+
+1. The rate of messages generated by a source (expressed in msg/s)
+2. Size of messages generated by a source
+
+### Performance parameters
+
+The performance parameters this tool records can be found in the [`Metrics`](https://github.com/ConduitIO/streaming-benchmarks/blob/b05577ea04932da48f0e620ed912b61d4c42d63b/main.go#L36) struct. The ones we use most when interpreting results are:
```
Comment (lovromazgon) [inline README.md:26]: Who needs line breaks 😶‍🌫️

**Review by lovromazgon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hariso) [reply README.md:26]: I'm not sure I understood.:)

Comment (lovromazgon) [reply README.md:26]: I'm saying the line has 245 characters, would be nice to break it up a bit 🙃

Comment (hariso) [reply README.md]: Ah, I didn't even notice it. I added this quickly through GitHub's web editor, and it appeared as new lines, but now I see that despite it's still the same line number.:)

**Pull request merged**